components/LinksHomepage/index.js

```diff
@@ -45,9 +45,7 @@ export default function LinksHomepage() {
     <>
       <StyledWrapper>
         <StyledForm onSubmit={handleSubmit}>
-          <StyledTitle>
-            <p>Add Recipe</p>
-          </StyledTitle>
+          <StyledTitle>Add Recipe</StyledTitle>
```

Comment:
The name does not refer to a `p` element: Use a heading here instead.

If this is the title of the form, ensure the a11y by using `aria-labelledby` attribute (or maybe test the screen reader first, it might also work because the heading is inside the form.